src/render_engine/collection.py

```diff
@@ -78,7 +78,11 @@ def __init__(
         self,
     ) -> None:
         if parser := getattr(self, "PageParser", None):
-            logging.warning(DeprecationWarning("PageParser is deprecated. Use Parser instead."))
+            logging.warning(
+                DeprecationWarning(
+                    "PageParser is deprecated. Use Parser from `render_engine_parser.base_parsers.BasePageParser`."
```

Comment:
When I mentioned a lack of detail I meant to highlight the object containing the Deprecation. This is important as if you’re using a custom Collection that you imported you may not know it’s using a deprecated attribute and then you would be looking for something that you can’t find.

```suggestion
                    f“The deprecated`PageParser` attribute is used in `{self.__class__.name}`. Use the `Parser` attribute instead."
```